Chase tests

chasewilson opened the pull request:
This is to add unit testing to the registry access resource. The integration testing for NTFS is still failing for an unknown reason. Please let me know if you have any questions or would like to see mor scenarios.

regedit32 commented:
I think we should take out the failing test for now and add it back in once it's resolved.

chasewilson commented:
Sounds good, I'll put them in as skipped so the test will still be there but won't run


Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Reggie Gibson <notifications@github.com>
Sent: Monday, October 23, 2017 8:30:14 AM
To: mcollera/AccessControlDsc
Cc: Chase Wilson; Author
Subject: Re: [mcollera/AccessControlDsc] Chase tests (#15)


I think we should take out the failing test for now and add it back in once it's resolved.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmcollera%2FAccessControlDsc%2Fpull%2F15%23issuecomment-338697900&data=02%7C01%7Cchase.wilson%40microsoft.com%7C664e646dd2994412830a08d51a2af59b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636443694164986836&sdata=mVaGdyLK%2Ff9u7w6aLUH%2FrHorAeLZ1sBfjsxecxJcZeE%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAd_xU0vNMaODX2_kpuZNbA4S8eig57hiks5svLEDgaJpZM4P6zYA&data=02%7C01%7Cchase.wilson%40microsoft.com%7C664e646dd2994412830a08d51a2af59b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636443694164986836&sdata=DOP81rUT1OIzotTeFzrBEHR56BaDaOUFcxAXu1Eg4Dk%3D&reserved=0>.

chasewilson commented:
I added the Pending switch and passed the build. It will still pass locally so I added a comment to remove the switch in local builds. Thanks for looking at it.
